test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -576,22 +576,6 @@
                         #"Found an old and unuspported database migration.*"
                         (migrate! *db*))))
 
-(deftest md5-agg-test
```

Comment:
this was a dupe - this test was in this file twice. I was getting confused running the tests.... "how could this fail? I swear I just fixed that one...."